documentation/modules/configuring/con-config-mirrormaker2-high-volume-messages.adoc

```diff
@@ -38,11 +38,7 @@ While this is happening, Kafka Connect tasks continue to poll topics in the sour
 The size of the buffer is set using the `producer.buffer.memory` property. 
 Tasks wait for a specified timeout period (`offset.flush.timeout.ms`) before the buffer is flushed. 
 This should be enough time for the sent messages to be acknowledged by the brokers and offset data committed. 
-
-NOTE: Kafka 3.1.0 and newer versions do not wait for the producer to empty the message queue before committing offsets, except during shutdown.
-
-Try to avoid a situation where there are too many messages in the buffer, so they can't all be flushed before the offset flush timeout is reached.
-The offset flush timeout period needs to be sufficient for the size of the buffer.
+Kafka does not wait for the producer to empty the message queue before committing offsets, except during shutdown.
```

Comment:
Instead of "Kafka does not wait", I would say the "The source task does not wait".